packages/hdwallet-core/src/ethereum.ts

```diff
@@ -102,6 +108,12 @@ export interface ETHWalletInfo extends HDWalletInfo {
    */
   ethSupportsNativeShapeShift(): boolean;
 
+  /**
+   * 
+   * Does the device support transactions with EIP-1559 fee parameters?
+   */
+  ethSupportsEIP1559(): boolean;
```

Comment:
I'd like to see this done up as a proper type guard... I'll do up an example

Reply:
(After discussion with @majorhayes and @mcchadwick, this will be handled as a fast-follow)